tests/DrupalConsoleStackTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
```

Comment:
Depending on whether we are going to use PSR-2 or PSR-12 this newline must respectively go or stay.